{
  translate_joy: true,
  new_button_size: 12,
  new_axes_size: 6,
  new_buttons: [1, 2, 0, 3, 4, 5, 8, 9, 10, 6, 7],
  new_axes: [0, 1, -1, 2, 3, -1, 4, 5],
  joy_trans_service: false
}

 